gradle.properties

```diff
@@ -1,15 +1,15 @@
 #group = org.opensearch.plugin.prometheus
 
 # An actual version of plugin
-version = 3.3.2.0
+version = 3.4.0.0
 
-opensearch_version = 3.3.2-SNAPSHOT
+opensearch_version = 3.4.0
```

Comment:
A real quick question: have you tried with `3.4.0-SNAPSHOT` ?

Reply:
Change it